test/Tester/StorageFacet/StorageFacetTests.cs

```diff
@@ -31,7 +32,8 @@ public ISiloBuilder CreateSiloBuilder(string siloName, ClusterConfiguration clus
                 {
                     ISiloBuilder builder = new SiloBuilder()
                         .ConfigureSiloName(siloName)
-                        .UseConfiguration(clusterConfiguration);
+                        .UseConfiguration(clusterConfiguration)
+                        .ConfigureLogging(loggingBuilder => TestingUtils.ConfigureDefaultLoggingBuilder(loggingBuilder, clusterConfiguration.GetOrCreateNodeConfigurationForSilo(siloName).TraceFileName));
```

Comment:
I think at this point we should introduce a base class or something. Although not sure @jason-bragg might have reservations, because he started there, and then moved away from it. Nevertheless the more stuff we move away from ClusterConfiguration, the more we will probably have to move to each test fixture, so it might make sense to have some defaults being defined somewhere else

Reply:
added to the issue list in #3429